util/file.go

```diff
@@ -189,10 +189,13 @@ func CopyFolderContentsWithFilter(source string, destination string, filter func
 			if err := CopyFile(file, dest); err != nil {
 				return err
 			}
+			if err := manifest.AddFile(dest); err != nil {
```

Comment:
The manifest file doesn't track folders? Just files?

Reply:
Yes, because on directories, it calls `CopyFolderContentsWithFilter` recursively, and subfolders will have their own manifests.

Reply:
Now it tracks directories as well.